examples/bsttest.wyv

```diff
@@ -0,0 +1,31 @@
+require stdout
+require java
```

Comment:
You don't need to require java when you're not using it below (it gets required implicitly anyway by stdout and binarysearchtree)